### DISCUSSION TITLE
No distriqt ANEs = make no sense to use AIR

### DISCUSSION THREAD

**Discussion by ghost** [airsdk/Adobe-Runtime-Support#1922] [General]
Hi, community! 
Please, don't get me wrong, but, honestly, all revenue from AIR apps is thanks to distriqt ANEs (AdMob, InAppPurchases, Firebase, etc). If distriqt tomorrow will stop supporting ANEs, then there is no sense to use AIR (despite free or demo apps), because there is no way to monetize apps. I don't no why, but Harman does not provide any of those. As for me - it really strange. This fact reduces confidence to Harman. We love AIR, but using AIR become unsafe for business nowadays. I am sorry for that.

**Comment by ajwfrost** [comment]
> I don't know why, but Harman does not provide any of those

Mostly we don't provide this because it's already provided by Distriqt :-)

It seems a bit harsh for us to develop our own versions of these, since that would mean we end up in competition with Distriqt. I prefer a kind of informal collaborative/complementary approach....

But yes, if Distriqt did decide to stop supporting the ANEs tomorrow, then we'd have to start providing these sorts of capabilities, because as you say it's a pretty crucial part of the ecosystem.

thanks

**Reply by elegantdeer** [reply]
Andrew 🙏🏻 We really thankful for Harman and your efforts. Long live for Distriqt 💖

**Comment by ydkgame** [comment]
Now you can choose to output the androidstudioproject project. Can you use any ane extension instead?
